packages/landing/components/navbar/Navbar.tsx

```diff
@@ -5,33 +5,55 @@ import MobileMenu from "./MobileMenu";
 import { ButtonLink } from "@/ui";
 import Logo from "./Logo";
 import { STACKLY_APP_URL } from "@/constants";
+import { useEffect, useRef, useState } from "react";
 
 export function Navbar() {
+  const buttonRef = useRef(null);
+  const [scrollY, setScrollY] = useState(0);
+  const [viewportHeight, setViewportHeight] = useState(600);
+  console.log("scrollY:", scrollY);
+
+  useEffect(() => {
+    if (window.innerHeight) setViewportHeight(window.innerHeight);
+  }, []);
```

Comment:
Shouldn't the "active button height" be fixed? Maybe we should show it after the user scrolls just a bit (100 pixels)?